modules/jupyter/main.tf

```diff
@@ -94,16 +109,26 @@ resource "helm_release" "jupyterhub" {
   values = [
     templatefile("${path.module}/jupyter_config/config-selfauth.yaml", {
       password            = var.add_auth ? "dummy" : random_password.generated_password[0].result
-      service_id          = var.add_auth ? (data.google_compute_backend_service.jupyter-ingress[0].generated_id != null ? data.google_compute_backend_service.jupyter-ingress[0].generated_id : "no-id-yet") : "no-id-yet"
-      backend_config      = var.service_name
+      project_id          = var.project_id
       project_number      = data.google_project.project.number
+
+      # Support legacy image.
+      service_id          = var.add_auth ? (data.google_compute_backend_service.jupyter-ingress[0].generated_id != null ? data.google_compute_backend_service.jupyter-ingress[0].generated_id : "no-id-yet") : "no-id-yet"
```

Comment:
this is not required after this change.?

Reply:
I assume the add auth is still optional?